.gitignore

```diff
@@ -22,3 +22,28 @@ logs/
 
 .ipynb_checkpoints
 ghostdriver.log
+
+#Pycharm IDE
```

Comment:
Can we not just remove `.idea`?

Reply:
At the moment I am using the .gitignore suggested by PyCharms - as this is a popular editor I would suggest leaving it as it is?